webapp/src/app/shared/phaser/helper.ts

```diff
@@ -65,32 +65,30 @@ export class Helper {
 	public static copy(obj: any) {
 		return JSON.parse(JSON.stringify(obj));
 	}
-	
+		
+
 	public static getJson(key: string, callback: (json: any) => void) {
 		const scene = Globals.scene;
-		
-		// get json from cache
+
```

Comment:
Cache will return old data. Consider including the map context in the scene cache key.

Reply:
Intended. The map editor never clears the cache so this is never an issue.